flocker/node/agents/blockdevice.py

```diff
@@ -498,7 +498,13 @@ def run(self, deployer):
         #     e2fsck refuses to run non-interactively, though.
         #
         # See FLOC-1814
-        check_output([b"e2fsck", b"-f", b"-y", device.path])
+
+        # The first call will exit with non-zero exit code as it needs to
+        # recreate lost+found. The second call ought to succeed.
+        command_line = [b"e2fsck", b"-f", b"-y", device.path]
+        call(command_line)
```

Comment:
This will write to stdout / stderr. Does that matter? Are we interested in the output? Will the output corrupt the logs sent to journald?